src/main/views/courts/tabs/generalContent.njk

```diff
@@ -70,14 +70,15 @@
       }
     ]
   }) }}
-
+{% set disableCommonPlatform = generalInfo.service_centre == true %}
 {{ govukCheckboxes({
     name: "common_platform",
     items: [
       {
         value: true,
         text: "Common Platform",
-        checked: generalInfo.common_platform
+        checked: generalInfo.common_platform,
+        disabled: disableCommonPlatform
```

Comment:
does it work if you did`disabled: generalInfo.service_centre` ?

Reply:
did try your suggestion throws of render page 
![Image 11-07-2022 at 17 33](https://user-images.githubusercontent.com/81159704/178313737-60d8ecba-95e4-4441-a50a-0e4fd53f3644.jpg)

Reply:
tried your suggestion and get page render error
![Image 11-07-2022 at 17 33](https://user-images.githubusercontent.com/81159704/178439021-3145c908-d9d9-4f08-819e-14df9282ab5e.jpg)

Reply:
updated the code to use your suggestion